spec/spec_helper.rb

```diff
@@ -25,25 +25,29 @@ def create_page_with_sourcebox iUrl=nil, subcards={},sourcebox=nil
   end
 end
 
-def create_claim_with_url name,url, subcards={}
+def create_claim_with_url name, url, subcards={}
   Card::Auth.as_bot do
     sourcepage = create_page url
-    Card.create! :type_id=>Card::ClaimID, :name=>name, 
-                 :subcards=>{ '+source' => {:content=>"[[#{sourcepage.name}]]",:type_id=>Card::PointerID}}.merge(subcards)    
+    Card.create! type_id: Card::ClaimID, :name=>name,
+                 subcards: {
+                   '+source' => {
+                     :content=>"[[#{sourcepage.name}]]",
+                     :type_id=>Card::PointerID
+                   }
+                }.merge(subcards)
   end
-
 end
 
 def create_claim name, subcards={}
   Card::Auth.as_bot do
     sourcepage = create_page 'http://www.google.com/?q=wikirateissocoolandawesomeyouknow'
-    Card.create! :type_id=>Card::ClaimID, :name=>name, 
-                 :subcards=>{ '+source' => {:content=>"[[#{sourcepage.name}]]",:type_id=>Card::PointerID}}.merge(subcards)    
+    Card.create! :type_id=>Card::ClaimID, :name=>name,
+                 :subcards=>{ '+source' => {:content=>"[[#{sourcepage.name}]]",:type_id=>Card::PointerID}}.merge(subcards)
```

Comment:
Line is too long. [122/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space missing after comma.<br>Surrounding space missing for operator `=>`.<br>Space inside } missing.<br>Space inside { missing.